bin/wetzel.js

```diff
@@ -5,12 +5,14 @@ var path = require('path');
 var argv = require('minimist')(process.argv.slice(2), { boolean : ["w", "suppresswarnings" ]});
 var defined = require('../lib/defined');
 var defaultValue = require('../lib/defaultValue');
-var generateMarkdown = require('../');
+var enums = require('../lib/enums');
+var generateMarkdown = require('../lib/generateMarkdown');
 
 if (!defined(argv._[0]) || defined(argv.h) || defined(argv.help)) {
     var help = 'Usage: node ' + path.basename(__filename) + ' [path-to-json-schema-file] [OPTIONS]\n' +
         '  -l,  --headerLevel        Top-level header. Default: 1\n' +
         '  -p,  --schemaPath         The path string that should be used when generating the schema reference paths.\n' +
+        '  -al, --autoLink           Aggressively auto-inter-link types referenced in descriptions.  Add =cqo to auto-link types that are in code-quotes only.\n' +
```

Comment:
Thanks for the catch.  I had forgotten to update the help after I did some validation and identified the same issue.  As far as I understand, with `minimist`, you can't have a single dash parameter with two letters -- the double-dash lets you specify multi-letter parameter names.  `l` was already taken for `headerLevel`...I can use `h` (for hyperlink) or just stick with `al` and just update the help/readme to indicate the double-dash is necessary.

Do you have an immediate preference?